roles/commcare_analytics/tasks/pg_backup.yml

```diff
@@ -0,0 +1,41 @@
+---
+- include_tasks: set_up_aws.yml
+  tags:
+    - postgres_backup
+    - aws_setup
+
+- name: Copy postgres credentials
+  become: yes
+  template:
+    src: "pgpass.j2"
+    dest: "{{ postgresql_user_home_dir }}/.pgpass"
+    group: "{{ postgresql_user }}"
+    owner: "{{ postgresql_user }}"
+    mode: 0600
+    backup: yes
+  tags:
+    - postgres_backup
+
+- name: Copy postgres backup script
+  become: yes
+  template:
+    src: create_postgres_dump.sh.j2
+    dest: /usr/local/bin/create_postgres_dump.sh
+    group: "{{ postgresql_user }}"
+    owner: "{{ postgresql_user }}"
+    mode: 0700
+    backup: yes
+  tags:
+    - postgres_backup
+
+- name: Create Weekly Cron job
```

Comment:
Should we not have a daily backup as well and be it customizable similar to [HQ](https://commcare-cloud.readthedocs.io/en/latest/backups_dr/2-backups-guide.html#postgresql-backups)

Reply:
My thought was to keep the weekly one for now and only add the daily one when we need it / want it in the future?